### PR TITLE
Updates for release 1.6.0

### PR DESCRIPTION
**Pull request by erwinvanhunen**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@get-itips Please advise.

**Comment by get-itips** [comment]
> @get-itips Please advise.

@erwinvanhunen Erwin is a PnP project maintainer so I think in this case it's ok to merge.

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Comment by Kateyanne** [comment]
@JohanFreelancer9 please copyedit. Thanks

File: sharepoint/sharepoint-ps/sharepoint-pnp/Add-PnPContentType.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Add-PnPContentType.md:41]:
```suggestion
This will add a new content type based on the standard document content type and assigns the document template template.docx to it.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Add-PnPContentTypesFromContentTypeHub.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Add-PnPContentTypesFromContentTypeHub.md:62]:
```suggestion
The list of content type IDs present in the content type hub site that are required to be added/synced to the current site.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Add-PnPTeamsTab.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Add-PnPTeamsTab.md:59]:
```suggestion
Specifies the URL to add.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Clear-PnpRecycleBinItem.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Clear-PnpRecycleBinItem.md:49]:
```suggestion
Permanently deletes the recycle bin item with Id 72e4d749-d750-4989-b727-523d6726e442 from the recycle bin.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Clear-PnpRecycleBinItem.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Clear-PnpRecycleBinItem.md:56]:
```suggestion
Permanently deletes the recycle bin item stored under variable $item from the recycle bin without asking for confirmation from the end user first.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Disable-PnPFlow.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Disable-PnPFlow.md:19]:
```suggestion
Disables a specific flow.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Get-PnPSyntexModel.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Get-PnPSyntexModel.md:45]:
```suggestion
Gets the content understanding model with ID 1.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Get-PnPSyntexModelPublication.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Get-PnPSyntexModelPublication.md:57]:
```suggestion
The name or ID of the SharePoint Syntex model.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPCompanyApp.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPCompanyApp.md:28]:
```suggestion
This cmdlet publishes a new company app (Microsoft Viva Connections) to Microsoft Teams. It will create a new package (zip file) in the current folder named after the CompanyApp value specified. For example, if the name is 'Contoso Portal', the package will be called "Contoso Portal.zip". This package will be uploaded to the Teams App Catalog. If you do not want to upload the package automatically, for example, prepare a package ahead of time, specify '-NoUpload'.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPCompanyApp.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPCompanyApp.md:87]:
```suggestion
The link to the public website of your company/organization.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPCompanyApp.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPCompanyApp.md:207]:
```suggestion
The Terms of Use link for the app. If not specified the default Terms of Use from the Microsoft link will be used.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPSyntexModel.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPSyntexModel.md:17]:
```suggestion
Publishes SharePoint Syntex models to a list.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPSyntexModel.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPSyntexModel.md:36]:
```suggestion
This command publishes a SharePoint Syntex content understanding model to a list.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPSyntexModel.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPSyntexModel.md:71]:
```suggestion
The name or ID of the SharePoint Syntex model.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPSyntexModel.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPSyntexModel.md:85]:
```suggestion
URL of the web hosting the list to publish the model to.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPSyntexModel.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPSyntexModel.md:100]:
```suggestion
The name or ID of the list to publish the model to.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPSyntexModel.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPSyntexModel.md:129]:
```suggestion
The server relative URL of the web hosting the library to publish the model to.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPSyntexModel.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPSyntexModel.md:144]:
```suggestion
The server relative URL of the library to publish the model to.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Remove-PnPTeamsTab.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Remove-PnPTeamsTab.md:46]:
```suggestion
Removes the tab with the display name 'Wiki' from the channel using ID.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Remove-PnPTeamsTab.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Remove-PnPTeamsTab.md:53]:
```suggestion
Removes a tab with the specified ID from the channel
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPHomeSite.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPHomeSite.md:39]:
```suggestion
Sets the home site to the provided site collection URL
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPHomeSite.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPHomeSite.md:58]:
```suggestion
The URL of the site to set as the home site
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPSite.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPSite.md:84]:
```suggestion
Allows custom script on a specific site. See [Allow or prevent custom script](/sharepoint/allow-or-prevent-custom-script) for more information.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPSite.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPSite.md:315]:
```suggestion
Specifies if a site allows custom script or not. See [Allow or prevent custom script](/sharepoint/allow-or-prevent-custom-script) for more information.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPTeamsTeamPicture.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPTeamsTeamPicture.md:39]:
```suggestion
Updates a picture for the team called 'MyTeam' with the available image at "c:\myimage.jpg"
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPTenant.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPTenant.md:930]:
```suggestion
For more information about sharing, see Manage external sharing for your SharePoint Online environment (/sharepoint/turn-external-sharing-on-or-off).
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Unpublish-PnPSyntexModel.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Unpublish-PnPSyntexModel.md:36]:
```suggestion
This command unpublishes a SharePoint Syntex content understanding model from a list.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Unpublish-PnPSyntexModel.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Unpublish-PnPSyntexModel.md:71]:
```suggestion
The name or ID of the SharePoint Syntex model.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Unpublish-PnPSyntexModel.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Unpublish-PnPSyntexModel.md:85]:
```suggestion
URL of the web hosting the list to unpublish the model from.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Unpublish-PnPSyntexModel.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Unpublish-PnPSyntexModel.md:100]:
```suggestion
The name or ID of the list to unpublish the model from.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Unpublish-PnPSyntexModel.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Unpublish-PnPSyntexModel.md:129]:
```suggestion
The server relative URL of the web hosting the library to unpublish the model from.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Unpublish-PnPSyntexModel.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Unpublish-PnPSyntexModel.md:144]:
```suggestion
The server relative URL of the library to unpublish the model from.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Add-PnPContentType.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Add-PnPContentType.md:41]:
```suggestion
This will add a new content type based on the standard document content type and assigns the document template template.docx to it.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/New-PnPTeamsTeam.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/New-PnPTeamsTeam.md:69]:
```suggestion
This will create a new Microsoft Teams team from an existing Microsoft 365 Group using the Group ID (teamify).
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPCompanyApp.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPCompanyApp.md:57]:
```suggestion
The path to the color icon (192x192px).
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPCompanyApp.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPCompanyApp.md:102]:
```suggestion
A short description for the app (less than 80 characters).
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPCompanyApp.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPCompanyApp.md:132]:
```suggestion
A longer description for the app (less than 4000 characters).
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPCompanyApp.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Publish-PnPCompanyApp.md:162]:
```suggestion
The path to the outline icon (32x32 px).
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Request-PnPSyntexClassifyAndExtract.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Request-PnPSyntexClassifyAndExtract.md:67]:
```suggestion
The name or list holding the files to classify and extract.
```

File: sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPHomeSite.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPHomeSite.md:74]:
```suggestion
[Set up a home site for your organization](/sharepoint/home-site)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by v-dihans** [comment]
Hi @erwinvanhunen

**Following are blocking issues that need to be resolved before we can merge:**

sharepoint/sharepoint-ps/sharepoint-pnp/Get-PnPMicrosoft365GroupMembers.md
sharepoint/sharepoint-ps/sharepoint-pnp/Get-PnPMicrosoft365GroupOwners.md
- Add an entry to the .openpublishing.redirection.json file for each of the deleted files. 
- Update the TOC to remove the deleted files.

sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPSite.md
- Lines 84, 315: We have new criteria that makes full (absolute) links to Microsoft docs articles blocking. Please make the links to Microsoft docs articles relative or site relative links.

sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPTenant.md
- Line 930: Full (absolute) links to Microsoft docs articles are blocking. Please make the links to Microsoft docs articles relative or site relative links.

&nbsp;
**Following are non-blocking issues that should be resolved now or within three business days in another PR:**

New articles
- If not already done, add a TOC entry for each new article

***

**Reference**

[Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master)

**Blocking**
- When an article is deleted, the PR includes a [redirect for the article in the .openpublishing.redirection.json](https://review.docs.microsoft.com/en-us/office-authoring-guide/retiring-redirecting-content?branch=master) file for the repo and an update to the TOC to remove the entry for the article.
- Links to other files in Microsoft Docs aren’t full-path URLs:
    - For links to files in the same doc set (folder), use file-relative links (include the file name extension).  Example: `safe-attachments.md`
    - For links to files in different doc sets (folders) or repos, use site-relative links (don't include the file name extension).
    - Example: `/microsoft-365/security/defender-365-security/safe-attachments`
    - Tip:   To get the site-relative link, remove`https://` from the beginning of the full path, and remove the file name extension.
    - For more information, see [Links](https://review.docs.microsoft.com/en-us/help/contribute/links-how-to?branch=master).


**Non-blocking**
- When a new article is added, the same PR must also contain the updated TOC with the entry for the new article. Exceptions are allowed when the file has **NOINDEX:NO FOLLOW** in the metadata.

***

**Questions/Comments**

sharepoint/sharepoint-ps/sharepoint-pnp/New-PnPTeamsTeam.md
- Line 69: The period at the end of the sentence displays in the diff view, but not on staging. Can you look into this?

sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPTenant.md
- Line 55: Entry displays as indented  [-EnableAutoNewsDigest <Boolean>]. Is this intentional?

sharepoint/sharepoint-ps/sharepoint-pnp/Unpublish-PnPSyntexModel.md
- Line 31: Entry displays as indented [-Connection <PnPConnection>]. Is this intentional?

Thanks

**Comment by JohanFreelancer9** [comment]
@Kateyanne, there appear to be eight suggested changes that were not committed. Could you please commit them as well? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by v-dihans** [comment]
Hi @erwinvanhunen

Thank you for fixing the absolute links. There are still blocking issues regarding the deleted files, and also a new issue from a recent commit. Once these issues are resolved we can merge the PR.

**Following are blocking issues that need to be resolved before we can merge:**

sharepoint/sharepoint-ps/sharepoint-pnp/Get-PnPMicrosoft365GroupMembers.md
sharepoint/sharepoint-ps/sharepoint-pnp/Get-PnPMicrosoft365GroupOwners.md
- Add an entry to the .openpublishing.redirection.json file for each of the deleted files. 
- Update the TOC to remove the deleted files.

sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPTenant.md
- Line 930: Fix the link. Markdown shows on staging. (It looks like you need brackets around the document name.)

&nbsp;
**Following are non-blocking issues that should be resolved now or within three business days in another PR:**

New articles
- If not already done, add a TOC entry for each new article

***

**Reference**

[Criteria used for pull request reviews](https://review./en-us/office-authoring-guide/pr-criteria?branch=master)

**Blocking**
- When an article is deleted, the PR includes a [redirect for the article in the .openpublishing.redirection.json](https://review./en-us/office-authoring-guide/retiring-redirecting-content?branch=master) file for the repo and an update to the TOC to remove the entry for the article.
- All links resolve correctly.
- The content preview must be clean on staging. It cannot contain any obvious formatting issues.

**Non-blocking**
- When a new article is added, the same PR must also contain the updated TOC with the entry for the new article. Exceptions are allowed when the file has **NOINDEX:NO FOLLOW** in the metadata.

***

**Questions/Comments**

sharepoint/sharepoint-ps/sharepoint-pnp/Set-PnPTenant.md
- Line 55: Entry displays as indented  [-EnableAutoNewsDigest <Boolean>]. Is this intentional?

sharepoint/sharepoint-ps/sharepoint-pnp/Unpublish-PnPSyntexModel.md
- Line 31: Entry displays as indented [-Connection <PnPConnection>]. Is this intentional?

Thanks

**Comment by yogkumgit** [comment]
@erwinvanhunen Gentle reminder.

**Comment by tiburd** [comment]
@tseward @get-itips @dariomws - Is this a PR that you should review and can help out with? If you cannot help or if you finish your review, please remove your ID from the **Assigned to** list. 

Please @ mention me in your response and let me know if you have any guidance or clarification. 

Thank you!

**Comment by get-itips** [comment]
Hello @erwinvanhunen do you still need this PR to get merged? As some time has passed and it might be outdated. If it is still valid please follow @v-dihans requests to make it valid for merge. Thank you

**Comment by yogkumgit** [comment]
@erwinvanhunen Gentle reminder

**Comment by tiburd** [comment]
@erwinvanhunen 
Hi, Erwin,
I'm closing this PR. You're welcome to open a new one.
Thanks!

**Pull request closed without merging**
